lib/whitehall_importer/create_image_revision.rb

```diff
@@ -56,7 +56,9 @@ def create_blob_revision(temp_image)
     def normalise_image(file)
       upload_checker = Requirements::ImageUploadChecker.new(file)
       abort_on_issue(upload_checker.issues)
-      normaliser = ImageNormaliser.new(file)
+
+      stripped_image = MiniMagick::Image.open(file.path).strip
```

Comment:
Does this create a new file or modify the existing file? just wondering with regard to the need of the file and/or if it modifies the fixture.

Reply:
It creates a new temp file so we have to pass in the resulting stripped image rather than relying on the original as the paths diverge.

Reply:
Great - glad it doesn't modify the source.